README.md

```diff
@@ -611,96 +602,69 @@ The full `options` object:
+Note that the `logo` and `section` property only apply to the "modal" form factor, and not the "inline".
 
-1. `"user"`: user closed the dialog via UI
-2. `"auto"`: the modal was closed because user successfully authenticated
-3. `"external"`: the application itself called the `modal.close` function
+### Links Configuration
 
-### Inline
+Use this configuration option to set the URL for your site or app's Terms & Conditions page. This supports the the `TermsAndConditionsCallback` that's commonly found in a registration flow.
 
-The named `form` import provides a simple `onMount` event.
+Example:
 
 ```js
-import { form } from 'forgerock-login-widget/inline';
+const config = configuration();
 
-// Know when the inline form has mounted
-form.onMount((formElement) => {
-  /* Run anything you want */
+config.set({
+  links: {
+    termsAndConditions: 'https://example.com/terms', // Full canonical URL for your app's terms and conditions page
+  },
 });
 ```
 
-It's worth noting that if the Widget has already mounted before the `onMount` statement, it will never run. It won't retroactively run the callback function.
+### Content Configuration
 
-### Styling API
-
-The Widget can be configured for styling purposes via the JavaScript API. This allows you to choose the type of labels used or providing a logo for the modal.
+This configuration setting is for passing custom content to the Widget, replacing its default content. For the full content schema, please [use the example en-US locale file](/src/locales/us/en/index.ts).
 
 Example:
 
 ```js
-const widget = new Widget({
-  target: document.getElementById('widget-root'),
-  props: {
-    config: {
-      /* ... */
-    },
-    content: {
-      /* ... */
-    },
-    /**
-     * OPTIONAL
-     */
-    style: {
-      checksAndRadios: 'animated', // OPTIONAL; choices are 'animated' or 'standard'
-      labels: 'floating', // OPTIONAL; choices are 'floating' or 'stacked'
-      logo: {
-        // OPTIONAL; only used with modal form factor
-        dark: 'https://example.com/img/white-logo.png', // OPTIONAL; used if theme has a dark variant
-        light: 'https://example.com/img/black-logo.png', // REQUIRED if logo property is provided; full URL
-        height: '300px', // OPTIONAL; provides additional controls to logo display
-        width: '400px', // OPTIONAL; provides additional controls to logo display
-      },
-      sections: {
-        // OPTIONAL; only used with modal form factor
-        header: false, // OPTIONAL; uses a modal "header" section that displays logo
-      },
-      stage: {
-        icon: true, // OPTIONAL; displays generic icons for the provided stages
-      },
-    },
-  },
+const config = configuration();
+
+config.set({
+  content: {}, // Custom content that overrides Widget default content
 });
 ```
 
-Note that the `logo` and `section` property only apply to the "modal" form factor, and not the "inline".
-
 ## Currently **unsupported**
```

Comment:
After the numerous times this question came up this week, I think it would be beneficial for us to state in this section that they *will* be supported in a future release. There was some confusion over what it would *eventually* support and what will likely never be supported.

Reply:
Good point, I used a different format for this.